### ISSUE TITLE
Upgrading to meteor 1.12.2 resulted in asking me to define @googleProtocol, @googleFonts and many more after giving the run command

### ISSUE THREAD

**Issue by Deepak2851** [meteor/meteor#13407]
Hey there. I was trying to upgrade my meteor version to 1.12.2 and facing this error. I have never used them in my application and it is asking me to define the variable @googleProtocol. Even after hardcoding this value, it then asks me to define another variable named @googleFonts. Then it asks for @vericalAllign and many more which will completely change my UI. Please help me

**Error**:
=> Started proxy.
=> Meteor 3.0.3 is available. Update this project with 'meteor update'.
=> Started MongoDB.
=> Errors prevented startup:

   While processing files with less (for target web.browser):
   client/lib/semantic-ui/definitions/globals/site.import.less:29: variable @googleProtocol is undefined
   error: variable @googleProtocol is undefined

   While processing files with less (for target web.browser.legacy):
   client/lib/semantic-ui/definitions/globals/site.import.less:29: variable @googleProtocol is undefined
   error: variable @googleProtocol is undefined

**Comment by leonardoventurini** [comment]
It's hard to give you a solution out of the bat, looks like you have `semantic-ui` included directly in your project right? You might need to check if the package you use to transpile `less` files is updated and if you need to provide any changes to the source files. Have you upgraded many packages so far?

**Comment by Deepak2851** [comment]
yes I have updated all the less files and all the packages but still getting the same error
